calm-integrations/ey-calm-migration-scripts/ey-recreate-categories.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# TODO- Add authentication
+# TODO- Add skip registration flag
+
+# pylint: disable= E402
+
+import gevent.monkey
+gevent.monkey.patch_all()
+import sys
+import logging
+import warnings
+import os
+import requests
+import json
+from IPython.frontend.terminal.embed import InteractiveShellEmbed
+
+from calm.common.config import init_config, get_config_dict, get_config
+cfg = init_config()
+
+from calm.common.flags import gflags
+gflags.FLAGS(sys.argv)
+
+from calm.lib.model.store.db import create_db_connection
+from calm.pkg.common.scramble import init_scramble
+from calm.lib.model.store.db_session import create_session, flush_session, set_session_type
+import calm.lib.model as model
+
+warnings.filterwarnings("ignore")
+log = logging.getLogger('cshell')
```

Comment:
1. Logger name should also be changed. 
2. Use logger instead of print everywhere.

Reply:
done